tests/unit/test_simulated_stack.py

```diff
@@ -334,6 +334,29 @@ def testGetVariantSet(self):
         for badId in self.getBadIds():
             self.verifyGetMethodFails(path, badId)
 
+    def testGetVariant(self):
+        # get a variant from the search method
+        referenceName = '1'
+        start = 0
+        dataset = self.backend.getDatasets()[0]
+        variantSet = dataset.getVariantSets()[0]
+        request = protocol.SearchVariantsRequest()
+        request.variantSetId = variantSet.getId()
+        request.referenceName = referenceName
+        request.start = start
+        request.end = 2**16
+        path = '/variants/search'
+        responseData = self.sendSearchRequest(
+            path, request, protocol.SearchVariantsResponse)
+        variant = responseData.variants[0]
+
+        # get 'the same' variant using the get method
+        path = '/variants'
+        responseObject = self.sendGetObject(
+            path, variant.id, protocol.Variant)
+        self.assertEqual(responseObject.referenceName, referenceName)
+        self.assertEqual(responseObject.start, start)
```

Comment:
Why can't we do a deep equality test here? The simulated variant should be exactly the same whichever way we get it, that's why we seed the RNG with the position. We should be able to directly compare the objects, i.e., ``variant1 == variant2``. Also, testing just the first one seems a bit wimpy --- set a page size of 10 (say) and do a GET for all of them?